zh_cn.lang

```diff
@@ -415,98 +415,119 @@ gui.xaero_category_enable_protection=保护：关
+gui.xaero_box_partial_y_teleportation=传送到路径点时，将 Y 坐标偏移 0.5 以避免穿过地毯状方块掉入虚空。否则，使用整数 Y 坐标。
+gui.xaero_delete_reached_deathpoints=删除到达过的死亡地点
+gui.xaero_box_delete_reached_deathpoints=当你靠近到4个方块内时自动删除死亡地点。 这要求死亡地点在游戏中或小地图上可见。 死亡地点在创建后5秒内不会被删除。
+gui.xaero_hide_minimap_under_screen=位于其他界面下方时隐藏
+gui.xaero_box_hide_minimap_under_screen=当GUI界面打开时隐藏小地图，例如物品栏，与小地图相关的界面和聊天界面除外。
+gui.xaero_north_compass_color=指南针北方颜色
 gui.xaero_north_compass_color_default=默认
 gui.xaero_category_add_to_list_custom=确认自定义输入：%1$s
-gui.xaero_box_category_list_add=支持带有特殊字符 * ) ( 和 | 的实体标识符或匹配模式。\n \n * - 匹配所有。 \n )( - 括号，主要用于 |. \n | - “或”运算符，以匹配相同位置的不同序列。 \n \n 例如，\n minecraft:creeper \n minecraft:zombie(_villager|_horse) - 僵尸村民和僵尸马。 \n minecraft:a*l - 任何以'a'开头、'l'结尾的原版实体。
+gui.xaero_box_category_list_add=支持带有特殊字符 * ) ( 和 | 的实体标识符或匹配模式。 \n \n * - 匹配任何内容。 \n ) ( - 括号，主要与 | 一起使用。 \n | - “或”运算符以匹配相同位置的不同序列。 \n \n 例如： \n minecraft:creeper \n minecraft:zombie(_villager|_horse) - 僵尸村民和马。 \n minecraft:a*l - 任何以“a”开头并以“l”结尾的原版实体。
 gui.xaero_using_custom_subworld=路径点：%1$s
-gui.xaero_hide_minimap_under_f3=F3下隐藏
-gui.xaero_box_hide_minimap_under_f3=启用了F3调试屏幕时，隐藏小地图。
-gui.xaero_entity_display_y=显示 Y 坐标
-gui.xaero_entity_display_y_full=显示实体的 Y 坐标
-gui.xaero_block_light_value=方块光：%d
-gui.xaero_sky_light_value=天空光：%d
-gui.xaero_all_light_value=光：%d
-gui.xaero_both_light_value=方块光：%d 天空光：%d
-gui.xaero_toggle_manual_cave_mode=启用手动洞穴模式
+gui.xaero_hide_minimap_under_f3=在 F3 时隐藏
+gui.xaero_box_hide_minimap_under_f3=启用 F3 调试界面时隐藏小地图。
+gui.xaero_entity_display_y=显示 Y 高度
+gui.xaero_entity_display_y_full=显示实体 Y 高度
+gui.xaero_block_light_value=方块光照：%d
+gui.xaero_sky_light_value=天空光照：%d
+gui.xaero_all_light_value=光照等级：%d
+gui.xaero_both_light_value=方块光照：%d 天空光照：%d
+gui.xaero_toggle_manual_cave_mode=切换手动洞穴模式
 gui.xaero_manual_cave_mode=手动洞穴模式
-gui.xaero_manual_cave_mode_start=手动洞穴模式起始点Y值
-gui.xaero_box_manual_cave_mode_start=启用了手动洞穴模式时，从这个Y值开始映射方块。手动洞穴模式可以用按键绑定启用。对于“自动”选项，一直走左边。
+gui.xaero_manual_cave_mode_start=手动洞穴模式起始 Y 高度
+gui.xaero_box_manual_cave_mode_start=启用手动洞穴模式时，从哪个Y高度开始绘制方块。 手动洞穴模式可以通过按键绑定进行切换。 对于“自动”选项，一直拉到左边。
 gui.xaero_manual_cave_mode_start_auto=自动
-gui.xaero_chunk_grid_line_width=区块网线条宽度
-gui.xaero_radar_render_over_minimap=在小地图框架之上渲染
-gui.xaero_box_radar_render_over_minimap=Render the entities over the minimap frame instead of rendering them inside the frame. Entities rendered this way act similarly to the waypoints and stick to the border when out of bounds. Visible entities are still limited to the ones loaded on the game client side though. \n \n Option "Player List" - render over minimap only when the "List Players" key binding (TAB by default) is pressed.
-gui.xaero_radar_render_radar_over_frame=在框架之上渲染雷达
+gui.xaero_chunk_grid_line_width=区块网格线宽度
+gui.xaero_radar_render_over_minimap=在小地图边框之上渲染
+gui.xaero_box_radar_render_over_minimap=在小地图边框上渲染实体，而不是在边框内渲染它们。 以这种方式渲染的实体的行为类似于路径点，并且在超出边界时会粘在边界上。 可见实体仍然仅限于游戏客户端加载的实体。 \n \n 选项“玩家列表” - 仅当按下“玩家列表”键绑定（默认为 TAB）时才在小地图上渲染。
```

Comment:
Does this mention the minimap border when talking about the Player List option? It should, so please make sure.

Reply:
Now it means more clearly

Reply:
Awesome!